CRM/Utils/PagerAToZ.php

```diff
@@ -151,7 +151,7 @@ public static function createLinks(&$query, $sortByCharacter, $isDAO) {
       $qfKey = CRM_Utils_Array::value('qfKey', $query->_formValues);
     }
     if (empty($qfKey)) {
-      $qfKey = CRM_Utils_Request::retrieve('qfKey', 'String', $this, FALSE, NULL, $_REQUEST);
+      $qfKey = CRM_Utils_Request::retrieve('qfKey', 'String', NULL, FALSE, NULL, $_REQUEST);
```

Comment:
@spalmstr i picked up on your fix through https://github.com/civicrm/civicrm-core/pull/10748 which monish has now merged, Thanks for the pointer